lsp/markdown/hover/enum.md

```diff
@@ -1,31 +0,0 @@
-## Type enums (sum types)
```

Comment:
Why did this and verb get deleted?

Reply:
The text is stored directly in the map in https://github.com/TBD54566975/ftl/pull/1835/files#diff-c73c1608401f635979f7387a53af47a0326467bc6d74a5a63b480af8eb444a48R4

I suppose it might be better to debug if it's human readable. Maybe a multiline string in the map.

Reply:
Oh sorry! I thought this was in the docs directory. Disregard.